lib/locales/el.ts

```diff
@@ -0,0 +1,501 @@
+import { Locale } from '../types/locale';
+
+export const locale: Locale = {
+  id: 'el',
+
+  seasons: {
+    advent: {
+      season: 'Περίοδος της Παρουσίας',
+      weekday: '$t(weekdays:{{dow}}) της $t(ordinals:{{week}}, { "context": "abbreviation" }) Εβδομάδας της Παρουσίας',
+      sunday: '$t(ordinals:{{week}}) Κυριακή της Παρουσίας',
+      privileged_weekday: '{{day}} $t(months:11)',
+    },
+
+    christmas_time: {
+      season: 'Περίοδος των Χριστουγέννων',
+      day: '$t(weekdays:{{dow}}) των Χριστουγέννων',
+      octave: '{{count}}η ημέρα της Ογδοάδας των Χριστουγέννων',
+      second_sunday_after_christmas: 'Δεύτερη Κυριακή μετά τα Χριστούγεννα',
+      before_epiphany: '{{day}} $t(months:0)',
+      after_epiphany: '$t(weekdays:{{dow}}) μετά την πανήγυρη των Θεοφανίων',
+    },
+
+    ordinary_time: {
+      season: 'Kοινή περίοδος του έτους',
+      weekday:
+        '$t(weekdays:{{dow}}) της $t(ordinals:{{week}}, { "context": "genitive" }) της κοινής περίοδου του έτους',
+      sunday: '$t(ordinals:{{week}}) Κυριακή της κοινής περίοδου του έτους',
+    },
+
+    lent: {
+      season: 'Περίοδος της αγίας Τεσσαρακοστής',
+      weekday:
+        '$t(weekdays:{{dow}}) της $t(ordinals:{{week}}, { "context": "abbreviation" }) Εβδομάδας της Τεσσαρακοστής',
+      sunday: '$t(ordinals:{{week}}) Κυριακή της Τεσσαρακοστής',
+      day_after_ash_wed: '$t(weekdays:{{dow}}) των Τεφρών',
+      holy_week_day: 'Μεγάλη $t(weekdays:{{dow}})',
+    },
+
+    paschal_triduum: { season: 'Πασχαλινό τριήμερο' },
+
+    easter_time: {
+      season: 'Περίοδος του Πάσχα',
```

Comment:
Incorrect characters (`ί` and `ά`).

```suggestion
      season: 'Περίοδος του Πάσχα',
```